packages/sources/coinmetrics/tsconfig.json

```diff
@@ -6,9 +6,5 @@
   },
   "include": ["src/**/*", "src/**/*.json"],
   "exclude": ["dist", "**/*.spec.ts", "**/*.test.ts"],
-  "references": [
-    { "path": "../../core/test-helpers" },
-    { "path": "../../core/bootstrap" },
-    { "path": "../../core/test-helpers" }
-  ]
+  "references": [{ "path": "../../core/test-helpers" }, { "path": "../../core/bootstrap" }]
```

Comment:
Think this slipped through the cracks with the `-test` adapter but we should remove this ref to core